core/lib/generators/solidus/update/update_generator.rb

```diff
@@ -57,6 +57,11 @@ def create_new_defaults_initializer
                File.join(options[:initializer_directory], "#{options[:initializer_basename]}.rb")
     end
 
+    def install_migrations
+      say_status :copying, "migrations"
+      rake 'railties:install:migrations'
```

Comment:
What about running `bin/rails railties:install:migrations FROM=spree` so that we don't get migrations from other engines?

Reply:
That's a valid concern, and I was thinking about it, but if there are pending migrations from other engines, why shouldn't we bring them in? I couldn't think of scenarios when it might be a problem.

Reply:
I think it's a matter of expectations. The Solidus update task should only deal with its stuff. We don't know in which scenario that task could be run, so I think it's good if we don't mess with other engines. Users might want to go step by step for any reason. What do you think?

Reply:
You are right, done!

Reply:
I randomly faced a similar problem in another context and realized how scoping the copy of migrations only for the spree namespace might be an issue: if solidus depends on another Rails engine, and solidus bumps it in a new version, and the new required version of that engine has new migrations, users will need to run the install migrations command for that engine or the system won't work. Do you think it's a realistic scenario?

Reply:
However, Solidus doesn't depend on any Rails engine with migrations, right? If so, we should then be explicit and copy the migrations from its namespace. What do you think?